test/e2e/upgrade_test.go

```diff
@@ -90,7 +90,7 @@ func TestUpgrade(t *testing.T) {
 		checkRecommendations(t, data, *upgradeToVersion)
 	}
 	// downgrade and check
-	applyNewVersion(t, data, upgradeFromAntreaYML, upgradeFromChOperatorYML, upgradeFromFlowVisibilityYML)
+	applyNewVersion(t, data, upgradeToAntreaYML, upgradeFromChOperatorYML, upgradeFromFlowVisibilityYML)
```

Comment:
Looks like this change is unrelated to TAD and has been covered in the https://github.com/antrea-io/theia/pull/383. There is also a `Golangci-lint` failure in line 38. Could we let that PR merge first and rebase this one?

Reply:
Yes, I was first trying to resolve the upgrade from N-1 test in this PR, but then created a new one. I will rebase this.